src/ansys/geometry/core/designer/component.py

```diff
@@ -266,6 +266,21 @@ def name(self) -> str:
         """Name of the component."""
         return self._name
 
+    @name.setter
+    def name(self, value: str):
+        """Set the name of the component."""
+        self.set_name(value)
+
+    @protect_grpc
+    @check_input_types
+    @min_backend_version(25, 2, 0)
+    def set_name(  # noqa: D102
+        self, name: str
+    ) -> None:
```

Comment:
Why no docstring? We should add it - this is a normal method. Remove the ``# noqa: D102`` comment and add the docstring.